packages/happo-target-firefox/src/index.js

```diff
@@ -0,0 +1,33 @@
+const { constructUrl } = require('happo-core');
+const runVisualDiffs = require('./runVisualDiffs');
+const initializeWebdriver = require('./initializeWebdriver');
+const closeDriver = require('./closeDriver');
+const checkBrowserVersion = require('./checkBrowserVersion');
+const server = require('./server');
+
+const DEFAULT_CONFIG = {
+  name: 'firefox',
+};
+
+module.exports = function firefox(config) {
+  const options = Object.assign({}, DEFAULT_CONFIG, config);
+
+  if (!options.name) {
```

Comment:
This is a fine type of check to have, but it seems a little pointless inside of the target plugin since it is pretty easy to see that the name is there. I think we would get more value out of this check at a higher level--whatever is consuming the target plugins.